### PR TITLE
Fix responsive margins on download buttons

### LINKED ISSUE #25: Buttons on releases page alignment on small screen devices
releases_buttons

### PR DESCRIPTION
**Pull request by Michadelic**
* moved left margin to right margin to have overflow buttons align to the left
* added small bottom margin to keep button distance on overflow

fixes #25

### CODE REVIEW

File: src/components/button/styles.css [hunk 1]
```diff
@@ -37,31 +37,33 @@
...
-.button.mod_download:not(:first-child) {
-	margin-left: 20px;
-	}
-	.button.mod_download:hover .button__icon g {
-		fill: #fff;
-		}
-	.button.mod_download:hover .button__version {
-		color: rgba(255, 255, 255, 0.5);
-		}
+	margin-bottom: 7px;
```
Comment (pavelkornev) [inline src/components/button/styles.css]: @Michadelic, I'm sorry, but now buttons are not aligned by vertical center in the block. This bottom margin should be compensated by inner block padding of a blue block:
incorrect_margin

File: src/components/button/styles.css [hunk 2]
```diff
@@ -37,31 +37,33 @@
...
+}
+.button.mod_download:not(:last-child) {
+	margin-right: 20px;
+}
+.button.mod_download:hover .button__icon g {
+	fill: #fff;
+}
+.button.mod_download:hover .button__version {
+	color: rgba(255, 255, 255, 0.5);
+}
```
Comment (pavelkornev) [inline src/components/button/styles.css]: Please fix also formatting of indentations in this file.

**Comment by pavelkornev** [comment]
Michael, please fix formatting of `src/components/button/styles.css`:
- there should be no white-space changes to unchanged lines of code
- CSS should look consistent with other rules/files in our project

**Comment by Michadelic** [comment]
@pavelkornev: could you please elaborate on the rules how this files is indented. I really can't see a 
proper indentation pattern in this file.

I usually follow this simply syntax in CSS, which i also did in this and all other changes:

.selector {
	property: value;
}

@media {
	.selector {
		property: value;
	}
}

**Comment by pavelkornev** [comment]
@Michadelic for CSS we have 2 simple things in this project:

- naming (please see — http://getbem.com/naming/)
- folding:
  - sub elements have to indented with 1 tabs from the left:
bem_basics
  - closing curly brace should be at the same position where the property name starts:
bem_basics_2

Please check other CSS files in this project where you can see all of them are following this simple pattern.

**BTW, I fixed code style in this PR for you.**

**Pull request merged**